modules/nw-ovn-ipsec-external.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-ovn-ipsec-external_{context}"]
+= IPsec encryption for external traffic
+
+{product-title} supports IPsec encryption for traffic to external hosts with TLS certificates that you must supply.
+
+This feature is supported on the following platforms:
```

Comment:
@yuvalk @anuragthehatter  Shall we need to mention GCP or its just they need RFE from DB later and mention there

Reply:
certainly need to add GCP to this list
if we're going to limit it at all..

the other thing that need to be addressed is the careful wording around k8s-nmstate support on gcp, where we support it, but only for IPsec.

Reply:
I mean, IMHO, we should not limit the platforms supported

Reply:
@yuvalk, every platform is completely supported, no caveats? For example, I see IBM cloud has this note:

> IPsec on {ibm-cloud-name} supports only NAT-T. Using ESP is not supported.

Reply:
@yuvalk 

> the other thing that need to be addressed is the careful wording around k8s-nmstate support on gcp, where we support it, but only for IPsec.

What does this mean, exactly? We support the NMState Operator for only this purpose? Or we support something about Kubernetes nmstate configurations on GCP? Or something else? Or is k8s-nmstate interchangeable with the NMState Operator?